### PR TITLE
[remoteconfig][beta] add doc

### PR DESCRIPTION
**Pull request by celenechang**
### What does this PR do?

Add doc about RemoteConfig feature

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Minimum Agent Versions

Are there minimum versions of the Datadog Agent and/or Cluster Agent required?

* Agent: vX.Y.Z
* Cluster Agent: vX.Y.Z

### Describe your test plan

Write there any instructions and details you may have to test your PR.

### Checklist

- [X] PR has at least one valid label: `bug`, `enhancement`, `refactoring`, `documentation`, `tooling`, and/or `dependencies`
- [X] PR has a milestone or the `qa/skip-qa` label

### CODE REVIEW

File: docs/remote_config.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+From the [Fleet Automation page](link-to-be-added), a policy can be created to enable a feature on an eligible scope. The Datadog Operator will then update the agents with the necessary configuration.
+
+
+## Prerequisites
+
+* Datadog Operator v1.7.0+
+
+## Enabling Remote Configuration
+
+Remote Configuration is disabled by default. To enable it using the [datadog-operator helm chart](https://github.com/DataDog/helm-charts/tree/main/charts/datadog-operator), in your `values.yaml` file make the following changes:
```
Comment (brett0000FF) [inline docs/remote_config.md:18]:
```suggestion
Remote Configuration is disabled by default. To enable it using the [datadog-operator Helm chart](https://github.com/DataDog/helm-charts/tree/main/charts/datadog-operator), in your `values.yaml` file make the following changes:
```

**Review by brett0000FF — APPROVED** [review]
Just a couple small suggestions for you to consider from Docs. 😄

**Comment by celenechang** [comment]
thanks @brett0000FF !

File: docs/remote_config.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Remote Configuration (beta)
+
+This feature was introduced in Datadog Operator v1.7.0 and is currently in beta.
+
+## Overview
+
+Remote Configuration in the Datadog Operator allows you to enable features in a Kubernetes cluster from Datadog.
+
+You can create a policy on the Fleet Automation page to enable a feature for an eligible scope. The Datadog Operator then updates the Agents with the necessary configuration.
```
Comment (levan-m) [inline docs/remote_config.md:9]: Should we reference some public doc here? It wouldn't be obvious what policy or scope means for people new to RC.
E.g. one of these or both
https://docs.datadoghq.com/agent/remote_config/
https://www.datadoghq.com/blog/remote-configuration-for-datadog/

File: docs/remote_config.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## Prerequisites
+
+* Datadog Operator v1.7.0+
+
+## Enabling Remote Configuration
+
+Remote Configuration is disabled by default. To enable it using the [datadog-operator Helm chart](https://github.com/DataDog/helm-charts/tree/main/charts/datadog-operator), in your `values.yaml` file make the following changes:
+
```
Comment (levan-m) [inline docs/remote_config.md:18]: maybe include something users would just copy, edit, paste?

File: docs/remote_config.md (hunk 2 above)
Comment (celenechang) [reply docs/remote_config.md:9]: We decided not to intentionally during this beta

**Review by levan-m — APPROVED** [review]

**Pull request merged**